app/src/androidTest/java/ch/epfl/skysync/HomeNavigationTest.kt

```diff
@@ -0,0 +1,62 @@
+package ch.epfl.skysync
+
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.navigation.compose.ComposeNavigator
+import androidx.navigation.compose.NavHost
+import androidx.navigation.testing.TestNavHostController
+import ch.epfl.skysync.navigation.Route
+import ch.epfl.skysync.navigation.homeGraph
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class HomeNavigationTest {
+
+  @get:Rule val composeTestRule = createComposeRule()
+  lateinit var navController: TestNavHostController
+
+  @Before
+  fun setUpNavHost() {
+    composeTestRule.setContent {
+      navController = TestNavHostController(LocalContext.current)
+      navController.navigatorProvider.addNavigator(ComposeNavigator())
+      NavHost(navController = navController, startDestination = Route.MAIN) {
+        homeGraph(navController)
+      }
+    }
+  }
+
+  @Test
+  fun verifyHomeIsStartDestination() {
+    val route = navController.currentBackStackEntry?.destination?.route
+    Assert.assertEquals(route, Route.HOME)
+  }
+
+  @Test
+  fun routeIsRightIfClickOnCalendar() {
+    composeTestRule.onNodeWithText("Calendar").performClick()
```

Comment:
I did not know this onNodeWithText function. This simplifies testing, but the text must be unique throughout the application to work properly.